satpy/etc/composites/abi.yaml

```diff
@@ -147,3 +147,261 @@ composites:
+    prerequisites:
+    - name: C03
+      modifiers: [sunz_corrected]
+    - name: C05
+      modifiers: [sunz_corrected]
+    - compositor: !!python/name:satpy.composites.DifferenceCompositor
+      prerequisites:
+        - name: C07
+        - name: C13
+    standard_name: snow
+
+  night_microphysics_abi:
+    ## RGB Nighttime Microphysics recipe source: http://rammb.cira.colostate.edu/training/visit/quick_guides/QuickGuide_GOESR_NtMicroRGB_final.pdf
+    compositor: !!python/name:satpy.composites.GenericCompositor
+    prerequisites:
+    - compositor: !!python/name:satpy.composites.DifferenceCompositor
+      prerequisites:
+        - name: C15
+        - name: C13
+    - compositor: !!python/name:satpy.composites.DifferenceCompositor
+      prerequisites:
+        - name: C13
+        - name: C07
+    - name: C13
+    standard_name: night_microphysics
+
+  fire_temperature_awips:
+    ## RGB composite for Fire Temperature source: http://rammb.cira.colostate.edu/training/visit/quick_guides/Fire_Temperature_RGB.pdf
+    ## adapted from etc/composites/viirs.yaml
+    compositor: !!python/name:satpy.composites.GenericCompositor
+    prerequisites:
+    - name: C07
+    - name: C06
+    - name: C05
+    standard_name: fire_temperature
+    name: fire_temperature_awips
+
+  land_cloud_fire:
+    ## RGB Day Land Cloud Fire recipe source: http://rammb.cira.colostate.edu/training/visit/quick_guides/QuickGuide_GOESR_DayLandCloudFireRGB_final.pdf
+    compositor: !!python/name:satpy.composites.GenericCompositor
+    prerequisites:
+    - name: C06
+    - name: C03
+      modifiers: [sunz_corrected]
+    - name: C02
+      modifiers: [sunz_corrected]
+    standard_name: land_cloud_fire
+
+  land_cloud:
+    ## RGB Day Land Cloud Fire recipe source: http://rammb.cira.colostate.edu/training/visit/quick_guides/QuickGuide_GOESR_daylandcloudRGB_final.pdf
+    compositor: !!python/name:satpy.composites.GenericCompositor
+    prerequisites:
+    - name: C05
+    - name: C03
+      modifiers: [sunz_corrected]
+    - name: C02
+      modifiers: [sunz_corrected]
+    standard_name: land_cloud
+
+  day_microphysics_eum:
+    ## adapted from etc/composites/ahi.yaml
+    ## source: http://www.eumetrain.org/rgb_quick_guides/quick_guides/DaymicroRGB.pdf
+    compositor: !!python/name:satpy.composites.GenericCompositor
+    prerequisites:
+    - name: C03
+      modifiers: [sunz_corrected]
+    - name: C07
+      modifiers: [nir_reflectance]
+    - name: C14
+    standard_name: day_microphysics
+
+  snow:
+    ## adapted from etc/composites/visir.yaml
+    ## source: http://www.eumetrain.org/rgb_quick_guides/quick_guides/SnowRGB.pdf
+    compositor: !!python/name:satpy.composites.GenericCompositor
+    prerequisites:
+    - name: C03
+      modifiers: [sunz_corrected]
+    - name: C05
+      modifiers: [sunz_corrected]
+    - name: C07
+      modifiers: [nir_reflectance]
+    standard_name: snow
+
+  day_microphysics_abi:
```

Comment:
Could you move this one above `snow` so it is just below the other `day_microphysics_eum`?

Reply:
It's ready